apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx

```diff
@@ -41,6 +43,20 @@ type RouteParams = {
 
 type TimeoutReturn = ReturnType<typeof setTimeout>;
 
+const FLOOR_PRICE_CURRENCIES = new Set(["ethereum"]);
+const getFloorPrice = async (nft: ProtoNFT, currency: Currency): Promise<number | null> => {
```

Comment:
This method should be in "live-common" as it's purely logic thing.
the base url `https://nft.api.live.ledger.com` should also be an env, so we can track all our envs consistently in "env.ts" of live-common.